app/helpers/pipeline_runs_helper.rb

```diff
@@ -4,7 +4,7 @@ module PipelineRunsHelper
   def aegea_batch_submit_command(base_command,
                                  memory = Sample::DEFAULT_MEMORY_IN_MB,
                                  job_queue: nil,
-                                 docker_image: "idseq_dag")
+                                 docker_image: "idseq_dag_meera1")
```

Comment:
Don't change the default here -- call the function with argument "idseq_dag_meera1" in your pipeline_run_stage command

Reply:
resolved